src/components/Map/MapRenderUtils.tsx

```diff
@@ -1,88 +1,9 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { useMap } from 'react-map-gl';
 
-import { Box, IconButton, Theme, Typography, useTheme } from '@mui/material';
-import { makeStyles } from '@mui/styles';
+import { Box, IconButton, Typography, useTheme } from '@mui/material';
 import { Button, Icon } from '@terraware/web-components';
 
-import strings from 'src/strings';
-import { MapPopupRenderer, MapSourceProperties } from 'src/types/Map';
-
-const useStyles = makeStyles((theme: Theme) => ({
-  popup: {
-    '& > .mapboxgl-popup-content': {
-      borderRadius: '8px',
-      padding: '10px',
-    },
-  },
-  button: {
-    marginLeft: theme.spacing(2),
-    '&:focus': {
-      outline: 'none',
-    },
-  },
-  tooltip: {
-    '& .mapboxgl-popup-content': {
-      borderRadius: theme.spacing(1),
-      padding: 0,
-    },
-    '& .mapboxgl-popup-close-button': {
-      display: 'none',
-    },
-  },
-}));
-
-export const mapTooltipDialogStyle = (theme: Theme) => ({
-  tooltip: {
-    '& .mapboxgl-popup-content': {
-      borderRadius: theme.spacing(1),
-      padding: 0,
-    },
-    '& .mapboxgl-popup-close-button': {
-      display: 'none',
-    },
-  },
-});
-
-/**
- * Species / plants renderer
- */
-export function useSpeciesPlantsRenderer(subzonesWithPlants: any): MapPopupRenderer {
```

Comment:
Is this not used?

Reply:
Surprisingly no, it was only referenced in one other file where it was just being imported and re-exported